docs/custom-transaction-name.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[custom-transaction-name]]
+=== Custom initial page load transaction names
+
+Correctly setting initial page load transaction names is vital to ensuring the best experience when viewing application performance metrics in the Kibana UI.
+
+Let's look at an example to understand why. Here are a few URLs you might find on elastic.co:
+
+[source,js]
+----
+// Blog Posts
+https://www.elastic.co/blog/reflections-on-three-years-in-the-elastic-public-sector
+https://www.elastic.co/blog/say-heya-to-the-elastic-search-awards
+https://www.elastic.co/blog/and-the-winner-of-the-elasticon-2018-training-subscription-drawing-is
+
+// Documentation
+https://www.elastic.co/guide/en/elastic-stack/current/index.html
+https://www.elastic.co/guide/en/apm/get-started/current/index.html
+https://www.elastic.co/guide/en/infrastructure/guide/current/index.html
+----
+
+By default, <<transaction-name,`transaction.name`>> is set to `unknown`. This means that when viewing transactions in the Kibana UI, all transactions will be grouped under the common transaction name of `unknown`. Grouping every transaction (especially unrelated ones) into one group doesn't make much sense. To fix this, you can use the API to change the default initial page load transaction name with the <<set-initial-page-load-name,`apm.setInitialPageLoadName()`>> method.
```

Comment:
It's actually `Unknown` with capital `U`, It might have been my own mistake, sorry about that!

Reply:
Gotcha! I wasn't sure of the difference between `setInitialPageLoadName` and `pageLoadTransactionName`. I've swapped them around!